### PR TITLE
Reorganized website

### PR DESCRIPTION
**Pull request by VickySteeves**
I added our new logo, updated the news page, added sectioning on the home page (about the news drop down), and our team photos.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -201,25 +216,31 @@
...
+			<tbody>
+				<tr>   
+					<td><img src="img/juliana.jpg" alt="Juliana Freire"/><p><a href="http://vgc.poly.edu/~juliana/">Juliana Freire</a></p></td>
+					<td><img src="img/dennis.jpg" alt="Dennis Shasha"/><p><a href="http://www.cs.nyu.edu/shasha/">Dennis Shasha</a></p></td>
+					<td><img src="img/remi.jpg" alt="Remi Rampin"/><p><a href="https://github.com/remram44"> Rémi Rampin</a></p></td>
+				</tr>
+				<tr>   
+					<td><img src="img/fernando.jpg" alt="Fernando Chirigati"/><p><a href="http://vgc.poly.edu/~fchirigati/">Fernando Chirigati</a></p></td>
+					<td><img src="img/vicky.jpg" alt="Vicky Steeves"/><p><a href="https://vickysteeves.com">Vicky Steeves</a></p></td>
+					<td><img src="img/heng.jpg" alt="Heng Li"/><p><a href="https://www.linkedin.com/in/heng2j">Zhonheng Li (summer 2017)</a></p></td>
```
Comment (remram44) [inline index.html]: I think the images should link to our homepages as well

File: news.html [hunk 2]
```diff
@@ -37,25 +37,38 @@
...
           <div class="navbar-header">
             <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-2">
               <span class="sr-only">Toggle navigation</span>
               <span class="icon-bar"></span>
               <span class="icon-bar"></span>
               <span class="icon-bar"></span>
             </button>
-            <a class="navbar-brand" href="index.html">ReproZip</a>
-          </div>
+			<a class="navbar-brand" href="#"><img alt="ReproZip" src="img/navbar-logo.png"></a>
```
Comment (remram44) [inline news.html]: It's awkward that this is a link to nowhere

**Comment by remram44** [comment]
Thanks! I like the new navbar! Should we roll this out first or do this and examples.reprozip.org together?

**Comment by remram44** [comment]
Bump

**Comment by VickySteeves** [comment]
Hey @remram44, I've made the following fixes:
+ changed footer to point to the new ViDA website
+ got rid of "back to top" on the index.html only
+ made our images click-able

We should probably roll this out at the same as the examples website, for consistency. I will push changes to the examples page soon.

File: news.html [hunk 3]
```diff
@@ -37,29 +37,42 @@
   </head>
 
   <body>
-    <div class="container" role="main">
 
+ <body>
```
Comment (remram44) [inline news.html]: double body

**Pull request merged**